### ISSUE TITLE
Add resources to Competitive Programming topic

### ISSUE THREAD

**Issue by Byte7** [vicky002/AlgoWiki#238]
Some possible suggestions are - sites which teach algorithms, sites which interactively visualize different algorithms. Also, notebooks/question pages of ACM themed contests can be added.

**Comment by rhcu** [comment]
Hi Byte7. Is this issue still relevant and needs help?

**Comment by Byte7** [comment]
@rhcu yes, the issue is still open. You can also contribute to any other domain, you might be interested in.

**Comment by shankarjavle01** [comment]
i want to work on this issue ..how can i get info about that ?

**Comment by PragatiVerma18** [comment]
@shankarjavle01, please contribute, this issue is open.

**Comment by WasdMash** [comment]
From personal experience, CodeWars is the best for absolute beginners who want to slowly develop over time. HackerRank can be used for the more advanced programmers

**Comment by AryanNayak** [comment]
Hey, is this issue open I would like to contribute?

**Comment by Byte7** [comment]
Hey @AryanNayak,
you can add to this issue.

**Comment by kendoriddy** [comment]
Hello, I would like to contribute to this issue.

**Comment by Byte7** [comment]
@kendoriddy Please make a fork and after adding, make a PR. We’ll review after that.

**Comment by xoxo16** [comment]
Hello , is this issue still not solved ?Can i work on it

**Comment by Vishwajeet-11** [comment]
https://github.com/vicky002/AlgoWiki/pull/378#issue-1104070025

**Comment by ghost** [comment]
@Byte7 is this issue still open? I would like to contribute.

**Comment by Byte7** [comment]
@prasadasabe Feel free to contribute.

**Comment by PratikshitGuptaI** [comment]
Is this issue still open to contribution? @Byte7

**Comment by DhirajGilda** [comment]
is this issue still open to contribute @Byte7

**Comment by sanskarsri** [comment]
Hey @Byte7 ,
Is this issue open? I would like to contribute can you please assign it to me?

**Comment by aniketghosh01** [comment]
Hey, @Byte7 I would like to contribute some books and resources related to Competitive Programming including the top-rated book "Competitive Programming 3 The New Lower Bound of Programming Contests" by Steven Halim. May I?

**Comment by ShubhamP48** [comment]
https://www.udemy.com/course/competitive-programming-algorithms-coding-minutes/
It is the best resourse out there,it covers all the  topics from basics like time complexity,spanning tress,dp etc.
it's paid but you can get it for less in sale and it's worth it....

**Comment by aniketghosh01** [comment]
@Byte7 Kindly review and merge my request

**Comment by raghav1030** [comment]
Hey there... Is this issue still open? It would be great if you assign it to me?

**Comment by abhi-kakadiya** [comment]
hey 
Is this Issue still open?,
I would like to Share Few Books and Links about Competitive Programming

**Comment by Ritika8081** [comment]
Can I work on this issue ?

**Comment by yeabsra2000** [comment]
what are the issues that i can contribute on

**Comment by Saanvi26** [comment]
@Byte7 is this issue still open? I would like to contribute.

**Comment by Mausoomi12** [comment]
@Byte7 Hey, is this issue still open? I'd like to make some contributions.
[Added c++ stl video resources](https://github.com/vicky002/AlgoWiki/pull/520)
